exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -103,6 +113,36 @@ impl JavaServiceRuntime {
     }
 }
 
+/// Returns path to <ejb-app location>/lib directory in an absolute form.
+fn absolute_library_path() -> String {
+    let library_path = {
+        let mut current_directory = executable_directory();
```

Comment:
installation_directory/executable_directory for the variable name — to avoid confusion with the working directory